app/src/main/java/com/android/unio/ui/explore/Explore.kt

```diff
@@ -78,83 +86,131 @@ fun ExploreScreenContent(
-              onQueryChange = { searchQuery.value = it },
-              onSearch = {
-                //                searchViewModel.searchAssociations(searchQuery.value)
-              },
-              expanded = false,
-              onExpandedChange = { /* Handle expanded state change here */},
-              placeholder = {
-                Text(
-                    text = "Search",
-                    style = AppTypography.bodyLarge,
-                    modifier = Modifier.testTag("searchPlaceHolder"))
-              },
-              trailingIcon = {
-                Icon(
-                    Icons.Default.Search,
-                    contentDescription = "Search icon",
-                    modifier = Modifier.testTag("searchTrailingIcon"))
-              },
-          )
-        },
-        expanded = false,
-        onExpandedChange = { /* Also handle expanded state change here */},
-        modifier = Modifier.padding(horizontal = 26.dp, vertical = 8.dp).testTag("searchBar"),
-        content = {},
-    )
-
-    LazyColumn(
-        modifier = Modifier.fillMaxSize().testTag("categoriesList"),
-        contentPadding = PaddingValues(vertical = 16.dp),
-        verticalArrangement = Arrangement.spacedBy(16.dp),
-    ) {
-      getSortedEntriesAssociationsByCategory(associationsByCategory).forEach {
-          (category, associations) ->
-        val alphabeticalAssociations = getFilteredAssociationsByAlphabeticalOrder(associations)
-
-        if (alphabeticalAssociations.isNotEmpty()) {
-          item {
-            Text(
-                text = category.displayName,
-                style = AppTypography.headlineSmall,
-                modifier =
-                    Modifier.padding(horizontal = 16.dp)
-                        .testTag("category_${category.displayName}"))
+  Column(
+      modifier = Modifier.padding(padding).fillMaxWidth(),
+      horizontalAlignment = Alignment.CenterHorizontally) {
+        Text(
+            text = "Explore our Associations",
+            /** Will go in the string.xml */
+            style = AppTypography.headlineLarge,
+            modifier =
+                Modifier.padding(top = 16.dp, bottom = 8.dp)
+                    .align(Alignment.CenterHorizontally)
+                    .testTag("exploreTitle"))
 
-            // Horizontal scrollable list of associations
-            LazyRow(
-                modifier =
-                    Modifier.fillMaxSize()
-                        .padding(vertical = 16.dp)
-                        .testTag("associationRow_${category.displayName}"),
-                contentPadding = PaddingValues(horizontal = 16.dp),
-                horizontalArrangement = Arrangement.spacedBy(32.dp, Alignment.Start),
-                verticalAlignment = Alignment.CenterVertically) {
-                  items(alphabeticalAssociations.size) { index ->
-                    AssociationItem(alphabeticalAssociations[index], navigationAction)
+        DockedSearchBar(
```

Comment:
Consider extracting this and the associated variables in a separate composable, as the `DockedSearchBar` is very big. This will reduce cognitive overload and make it easier to test this.